iceoryx2-bb/cxx/CMakeLists.txt

```diff
@@ -29,9 +29,13 @@ include(Iceoryx2PlatformSettings)
 add_library(iceoryx2-bb-containers-cxx INTERFACE)
 add_library(iceoryx2-bb-cxx::iceoryx2-bb-containers-cxx ALIAS iceoryx2-bb-containers-cxx)
 
+# TODO is this required? We install the full include directory anyway and
+# the natvis file has the build path in the iceoryx2-bb-cxxTargets.cmake file,
+# which probably will cause issues on Windows since I was not able to restrict
+# it to the BUILD_INTERFACE
```

Comment:
Btw, what was the problem with the natvis file you are referring to here? If I wrap the natvis source in a `$<BUILD_INTERFACE:>` generator expression, it seems to work as expected for me.

Reply:
Okay, I need to give it another try. Was already late when I tried it. But Windows also did not complain about paths, so this should be fine anyway.

Reply:
No, it is not essential. I would leave that for a follow-up task, as there is a number of possible ways how to include it, if desired.